examples/platform/silabs/provision/ProvisionStorageCustom.cpp

```diff
@@ -14,9 +14,9 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-#include "ProvisionStorage.h"
 #include <algorithm>
 #include <lib/support/CodeUtils.h>
+#include <platform/silabs/provision/ProvisionStorage.h>
```

Comment:
```suggestion
#include <ProvisionStorage.h>
```

Reply:
This is getting modified in @mkardous-silabs ongoing draft PR though so you can ignore it for now.

Reply:
I had that before but @mkardous-silabs suggested how it is written now. I don't really care either way.